### ISSUE TITLE
Review WCAG pagina's: 'opruimsprint'

### ISSUE THREAD

**Issue by RenateRoke** [nl-design-system/documentatie#3864]
## TLDR

@JuliaTol-properaccess heeft inmiddels een hoop WCAG pagina's ter review klaargezet. Zij loopt die zelf nog een keer aan de hand van de schrijfwijzer en ik zal er doorheen lopen op inhoud, voordat @Robbert de laatste reviews doet.  

Julia geeft in Slack bij mij aan welke pagina's er klaar staan voor review.

## Acceptatiecriteria

Julia heeft feedback gekregen op de set WCAG pagina's die zij klaar heeft gezet.
